### DISCUSSION TITLE
Router

### DISCUSSION THREAD

**Discussion by Bhushan0001** [vercel/next.js#69835] [App Router]
which is better for routing page router or App router

**Comment by hyjoong** [comment]
When starting a new project, it's recommended to use the App Router

**Reply by Bhushan0001** [reply]
Thank you !!!

**Comment by Juneezee** [comment]
My advice is the same as https://github.com/vercel/next.js/discussions/69835#discussioncomment-10580355.

While the Next.js team plans to support Pages Router for the next few years (https://github.com/vercel/next.js/discussions/56655#discussioncomment-8791401), the newest features of React such as Server Components, Server Actions, etc. are only available in App Router.

**Reply by Bhushan0001** [reply]
Thank you !!!
